site/pages/pledge/[address].tsx

```diff
@@ -0,0 +1,38 @@
+import { GetStaticProps } from "next";
+
+import { Pledge } from "components/pages/Pledge";
+import { getPledge } from "queries/pledge";
+import { loadTranslation } from "lib/i18n";
+
+export const getStaticProps: GetStaticProps = async (ctx) => {
+  const translation = await loadTranslation(ctx.locale);
+  const { address } = ctx.params as { address: string };
+  let pledge;
+
+  try {
+    const response = await getPledge(address.toLowerCase());
```

Comment:
Better if we use `getPledgeByAddress` here instead, to share the server-side code and avoid an extra network roundtrip, as recommended in the docs:
https://nextjs.org/docs/basic-features/data-fetching/get-static-props#write-server-side-code-directly